When running ./build.sh I get an error

# What's the issue:

I did some changes and wanted to build the project on my computer to test it.
Followed the documentation so everything was fine, including the env variables were set (just with empty strings for now).

When I executed `./build.sh` I get this error:

```bash
Building Mac binary...
cmd/appix/main.go:12:2: cannot find package "github.com/Travix-International/appix" in any of:
        /Users/RMachado/golang/src/Travix-International/appix/vendor/github.com/Travix-International/appix (vendor tree)
        /usr/local/Cellar/go/1.6.2/libexec/src/github.com/Travix-International/appix (from $GOROOT)
        /Users/RMachado/golang/src/github.com/Travix-International/appix (from $GOPATH)
cmd/appix/main.go:13:2: cannot find package "github.com/Travix-International/appix/config" in any of:
        /Users/RMachado/golang/src/Travix-International/appix/vendor/github.com/Travix-International/appix/config (vendor tree)
        /usr/local/Cellar/go/1.6.2/libexec/src/github.com/Travix-International/appix/config (from $GOROOT)
        /Users/RMachado/golang/src/github.com/Travix-International/appix/config (from $GOPATH)
```

Also, in case you're wondering, here's the values of the variables `$GOPATH` and `$GOROOT`:

```bash
➜  appix git:(proposal-ignore-strings-contains) ✗ echo $GOPATH
/Users/RMachado/golang
➜  appix git:(proposal-ignore-strings-contains) ✗ echo $GOROOT

```

Also, I had to comment the other OS'es builds 'cause it was triggering errors as well.

--- a/README.md
+++ b/README.md
@@ -51,7 +51,7 @@ For developing the ADK itself:
 Clone the repo to your `$GOPATH`:
 
 ```
-$ git clone git@github.com:Travix-International/appix.git $GOPATH/src/Travix-International/appix
+$ git clone git@github.com:Travix-International/appix.git $GOPATH/src/github.com/Travix-International/appix
 ```
 
 ### Dependencies

